pages/glossary/filename.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Filename
+key: filename
+---
+
+A filename is a text string that identifies an electronically stored file. It is commonly used as part of a URL path to identify one piece of content from another. For example the `src` attribute specifies a URL path of `src="/foo/bar.jpg?baz "` which contains the filename `bar.jpg`.
```

Comment:
This doesn't actually define how to determine the filename given a URL.

Reply:
What is the purpose of the definition?